unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadWrapper/Tests/LoadWrapperShould.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections;
+using DCL;
+using DCL.Components;
+using DCL.Helpers;
+using DCL.Models;
+using NSubstitute;
+using Tests;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Assert = UnityEngine.Assertions.Assert;
+using Object = UnityEngine.Object;
+
+public class LoadWrapperShould : IntegrationTestSuite
+{
+    protected override PlatformContext CreatePlatformContext() { return DCL.Tests.PlatformContextFactory.CreateWithGenericMocks(WebRequestController.Create()); }
+
+    [UnityTest]
+    public IEnumerator UnloadWhenEntityDestroyedBeforeFinishing()
+    {
+        GameObject meshRootGameObject = new GameObject();
+
+        string url = TestAssetsUtils.GetPath() + "/GLB/Lantern/Lantern.glb";
```

Comment:
```suggestion
        string url = TestAssetsUtils.GetPath() + "/GLB/Lantern/Lantern.glb";
```

The Lantern glb is pretty heavy, maybe we can make do with another model that takes less time to load?